cmd/cstor-volume-mgmt/volume/volume.go

```diff
@@ -196,6 +198,57 @@ func CreateIstgtConf(cStorVolume *apis.CStorVolume) ([]byte, error) {
 	return dataBytes, nil
 }
 
+// ResizeTargetVolume sends resize volume command to istgt and get the response
+func ResizeTargetVolume(cStorVolume *apis.CStorVolume) error {
+	index, _, err := FileOperatorVar.GetLineDetails(util.IstgtConfPath, "LUN0 Storage")
+	if err != nil {
+		return errors.Wrapf(err,
+			"failed to get storage details from %s file",
+			util.IstgtConfPath)
+	} else if index == -1 {
+		return errors.Errorf("storage details not found in file %s", util.IstgtConfPath)
+	}
+	// send resize command to istgt and read the response
+	resizeCmd := getResizeCommand(cStorVolume)
+	sockResp, err := UnixSockVar.SendCommand(resizeCmd)
+	if err != nil {
+		return errors.Wrapf(
+			err,
+			"failed to execute istgt %s command on volume %s",
+			util.IstgtResizeCmd,
+			cStorVolume.Name)
+	}
+	if v1_strings.MakeList(sockResp...).Contains("ERR") {
+		err = errors.Errorf("%v", sockResp)
+		return errors.Wrapf(
+			err,
+			"failed to execute istgt %s command on volume %s",
+			util.IstgtResizeCmd,
+			cStorVolume.Name)
+	}
+
+	updateStorageVal := fmt.Sprintf("  LUN0 Storage %s 32K", cStorVolume.Spec.Capacity.String())
+	err = FileOperatorVar.Updatefile(util.IstgtConfPath, updateStorageVal, index, 0644)
+	if err != nil {
+		return errors.Wrapf(err,
+			"failed to update %s file with %s details",
+			util.IstgtConfPath,
+			updateStorageVal)
+	}
+	glog.Infof("Updated '%s' file with capacity '%s'", util.IstgtConfPath, updateStorageVal)
+	return nil
+}
+
+// getResizeCommand returns resize used to resize volumes
```

Comment:
provide the sample command for resize in comments

Reply:
Done